setup.cfg

```diff
@@ -70,7 +70,7 @@ console_scripts =
 lookups =
     spacy_lookups_data>=1.0.2,<1.1.0
 transformers =
-    spacy_transformers>=1.0.1,<1.1.0
+    spacy_transformers>=1.1.0,<1.2.0
```

Comment:
I don't think we need to require this? People may have reasons to want to continue using v1.0 models and it's not strictly necessary?

Reply:
Technically we don't need to require it no. I do think the IO was pretty broken before 1.1.0 so it would be better if people upgraded, but we don't have to enforce it, it'll happen naturally.